### ISSUE TITLE
paper-chips autocomplete dropdown is rendered off-screen on long pages

### ISSUE THREAD

**Issue by wahhaj** [miguelcobain/ember-paper#740]
I have a form with a lot of input fields (the form is ~2000px tall), with a `paper-chips` field near the bottom that renders its autocomplete dropdown off the bottom of the viewport. 

If I move the `paper-chips` field higher up in the form, it works fine; the bug only occurs when you have to scroll down to get to it. If you try and scroll down to the off-screen dropdown, it keeps shifting down and you can never see it.

Here's a video: https://streamable.com/hcdx5

Here's a repo with the code for the example above. 
https://github.com/wahhaj/paper-chips-test

**Comment by miguelcobain** [comment]
What version of ember-paper are you using?
Also, are you using ember-basic-dropdown por ember-power-select in your app?

**Comment by miguelcobain** [comment]
Answered my tão questions with the repo you made.

**Comment by wahhaj** [comment]
Yes, that information is available in the test repo but to save anyone else reading this some time:

ember-paper version is **1.0.0-alpha.20**, and neither ember-basic-dropdown nor ember-power-select were manually added to the app.

**Comment by bltavares** [comment]
Would you mind trying out the following `css` change?

```css
body {
  position: initial;
}
```

I recently had as similar problem when upgrading to `1.0.0-beta.1` and so far on my debugging, this is due to a style that `angular-material` imported sass rules add to the body. The problem was actually on `paper-select` and `paper-autocomplete`, but it seems to be similar enough.

I do not know why they added styles to the `body` tag, but that change was enough for the wormwhole calculate correctly the position of the elements.

**Comment by lucasmerat** [comment]
@miguelcobain I am on the most recent version of ember-paper and am seeing this issue. I have attached an image. Am I incorrect in thinking that the dropdown should be responsive when at the bottom of the page and move to be visible and clickable to the user?
<img width="451" alt="Screen Shot 2020-08-17 at 11 19 34 AM" src="https://user-images.githubusercontent.com/32575355/90419087-a611b900-e07b-11ea-97c5-70d5817b21b0.png">

**Comment by JonnoFTW** [comment]
@bltavares Your fixed worked for me. Not sure if there will ramifications for any other css though.
